LondonBritishLibrary/orient/BLorient539.xml

```diff
@@ -0,0 +1,324 @@
+                        
+                        <additions>
+                            <list>
+                                <item xml:id="a1" corresp="#h2">
+                                        <locus from="224v" to="226v"/>
+                                    <desc type="GuestText"><title type="complete" ref="LIT3052RepCh332"/></desc>
+                                    <q xml:lang="gez"><seg part="I">እምሥርወ፡ ዕሴይ፡ እስመ፡ ሠረፅከ፡ ወእማኅፀና፡ ለሩት። ናስተበፅዓከ፡ ዘልፈ፡ በንባበ፡ ዝንቱ፡ ስብሐት። 
+                                        ኦብፁዕ፡ ንጉሠ፡ እስራኤል፡ ዳዊት። ሶበ፡ ተከሥተ፡ ወተረከበ፡ ባሕርየ፡ ውሣጣይ፡ ሕይወት። እመዝገበ፡ ከርሣ፡ ለማርያም፡ ዘለከ፡ ሕይወት። ሰላም፡ ለዝክረ፡ 
+                                        ስምከ፡ ውስተ፡ ረቀ፡ መዝሙር፡ ዘተዌደደ።
+                                   </seg></q>
+                                 </item>
+                                
+                                <item xml:id="a2">
+                                    <locus target="#226v"/>
+                                    <desc type="OwnershipNote">A note gave the name of a later owner, for which that of 
+                                        <persName ref="PRS13080Isayeyyas"/> has been substituted.</desc>
+                                    <q xml:lang="gez"/>
+                                </item>
+                                
+                                <item xml:id="a3">
+                                    <locus target="#227r"/>
+                                    <desc type="Unclear">Note in Amharic.</desc>
+                                    <q xml:lang="am"/>
+                                </item>
+                                
+                                <item xml:id="e1">
+                                    <locus target="#2v"/>
+                                    <desc>The names originally mentioned in the manuscript have been erased, 
+                                        only that of <persName ref="PRS13081WalattaH"/> is still legible.</desc>
+                                </item>
+                                
+                                <item xml:id="e2">
+                                    <locus target="#82r"/>
+                                    <desc>The names originally mentioned in the manuscript are partially legible. One ended by 
+                                    <foreign xml:lang="gez">ክርስቶስ></foreign>, which has been changed into 
+                                        <persName ref="PRS13079HellaweK"/>.</desc>
+                                </item>
+                                
+                                <item xml:id="e3">
+                                    <locus target="#2v"/>
+                                    <desc>The names of <persName ref="PRS13082KidanaM"/> and <persName ref="PRS13080Isayeyyas"/>
+                                        have been substituted for the 
+                                        erased names originally mentioned in the manuscript.</desc>
+                                </item>
```

Comment:
sorry, th description is not very clear to me: those names are substituitions for the original erased names?

Reply:
yes, please feel free to reformulate if you come up with something clearer.
The names originally mentioned in the manuscript have been erased, and substituted by...?

Reply:
Yes, I think that would be clearer, at least for me :)

Reply:
thank you!